README.md

```diff
@@ -58,6 +58,7 @@ The options are:
 * `fd`: a file descriptor, something that is returned by `fs.open` or
    `fs.openSync`.
 * `dest`: a string that is a path to a file to be written to (mode `'a'`).
+* `mkdir`: ensure directory for dest file exists.
```

Comment:
```suggestion
* `mkdir`: ensure directory for dest file exists when `true`. Default: `false`.
```

Reply:
Thanks for the suggestion. I'll apply it when I update the MR.